<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>服务器</string>
  </property>
  <widget class="QListWidget" name="listWidgetLOG">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>271</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditRECEIVE">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>接收</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2SEND">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>221</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>216</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditPORT">
      <property name="placeholderText">
       <string>端口号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>开启监听</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
